doc/content/verification_validation_examples/slab_benchmark/slab_benchmark.md

```diff
@@ -0,0 +1,291 @@
+
+\begin{equation}
+  \phi(x) = \phi_0 \sqrt{1-\frac{(\lambda-1)P^2x^2}{L^2q^2\phi^2_0}},
+  \label{eqn:flux_solution}
+\end{equation}
+
+and the temperature distribution is given by
+
+\begin{equation}
+  T(x) = \frac{q\Sigma_{t,0}T_0L}{P} \phi(x) ;.
+  \label{eqn:temperature}
+\end{equation}
+
+Note that [eqn:flux_solution] is identical to the formula for an ellipse rather than the cosine shape normally seen with constant cross sections in finite slab geometries, as a result of the benchmark physics coupling.
+
+The solution for the eigenvalue, $\lambda$, is given by
+
+\begin{equation}
+  \lambda = \frac{1}{2} \left(1+\sqrt{1+\frac{16q^2\phi^2_0}{P^2}} \right).
+  \label{eqn:eigenvalue}
+\end{equation}
+
+Using manufactured parameters for the benchmark specifications (e.g., $L_0$, $P$), the values for the microscopic total cross section $\sigma_{t,0}$ and convective heat transfer coefficient $h$ are determined such that the temperature distribution also satisfies the manufactured scenario where the flux and temperature distributions have the same shape as given by [eqn:temperature].
+
+The final length of the heated slab was evaluated using the flux and temperature solutions [eqn:flux_solution],[eqn:temperature], which were substituted into the mass conservation equation, [eqn:mass_conservation]. Integrating both sides gives the total length of the heated slab implicitly by preserving the total mass from the initial to the final state resulting
+
+\begin{equation}
+  \rho_0\int_{-L/2}^{L/2}\sqrt{\frac{T_0}{T(x)}}dx=\rho_0L_0.
+\end{equation}
+
+The final length of the heated slab $L$ is then given by
+
+\begin{equation}
+  L = \sqrt[3]{\Bigg(\frac{L_0}{_2F_1(1/4,1/2,3/2,1/\lambda)}\Bigg)^4\Bigg(\frac{\frac{q^2\phi^2_0}{P(\lambda-1)}-\frac{P}{4}}{\kappa_0T_0^2(1-1/\lambda)}\Bigg)}
+  \label{eqn:length},
+\end{equation}
+
+where $_2F_1$ is the Gauss hypergeometric function.
+
+To ensure a physically meaningful solution (i.e., the heat flux leaving the slab is consistent with the $T(\pm L/2)-T_0$), there is an additional constraint on the manufactured temperature distribution
+
+\begin{equation}
+  T\left(\pm\frac{L}{2}\right)\geq T_0.
+  \label{eqn:constraint}
+\end{equation}
+
+Substituting [eqn:flux_solution] and [eqn:temperature] into [eqn:constraint] gives a minimum condition for power in terms of the other slab parameters, as
+
+\begin{equation}
+  P \geq \frac{4\kappa_0T_0^2}{L(\lambda-1)}.
+  \label{eqn:power_condition}
+\end{equation}
+
+The example set of model parameter values shown in [tab:params] are the suggested benchmark parameters [!citep](griesheimer2022analytical), resulting in $k_\text{eff} = 0.29557$ and a heated slab length of $L = 106.47$ cm with $P = 1.0\times10^{22}$ eV/s.
+
+!table id=tab:params caption=Example set of model parameters with two different power scenarios [!citep](griesheimer2022analytical).
+| **Parameter                  | **Value                   | **Unit                              |
```

Comment:
Why do you use the `**` symbols everywhere? What do they do? 
I also don't see the need for the ` signs around the units.